packages/dx-react-scheduler/src/plugins/date-navigator.jsx

```diff
@@ -27,11 +29,11 @@ export class DateNavigator extends React.PureComponent {
 
     this.handleToggle = this.handleToggle.bind(this);
     this.handleHide = this.handleHide.bind(this);
-    this.buttonRef = this.buttonRef.bind(this);
+    this.targetRef = this.targetRef.bind(this);
```

Comment:
This method doesn't have a verb in it.

Reply:
@kvet should we rename [the similar function](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-grid/src/plugins/column-chooser.jsx#L22) in React Grid?

Reply:
Hm, I think we should